### DISCUSSION TITLE
setup.py gone, is pip the only way to install now?

### DISCUSSION THREAD

**Discussion by Mycah** [beetbox/beets#5313] [Q&A] [answered]
Just did a git pull for the first time in awhile and it appears that setup.py is gone, but it's still referenced in the docs.

Was this intentional?  Is the only way to build using pip now?

Thanks!

**Comment by Serene-Arc** [comment, marked as answer]
Hi, if you pull from GitHub, you need to select `latest` in the doc type. The project has switched to using poetry or pip/pipx to install now.

**Reply by Mycah** [reply]
Cool, thanks.  Is there somewhere that documents the poetry method?

The latest docs still show setup.py as a valid method of installing: https://beets.readthedocs.io/en/latest/faq.html#run-the-latest-source-version-of-beets

2nd item on the list, then 3rd bullet point.

**Reply by Mycah** [reply]
never mind, found it in the rst

https://github.com/beetbox/beets/blob/master/docs/faq.rst#run-the-latest-source-version-of-beets

**Reply by Serene-Arc** [reply]
Hmm, that's interesting. Perhaps it hasn't updated yet to the site. Thanks for bringing this to my attention.
